app/views/games/dashboard.html.erb

```diff
@@ -1,111 +1,90 @@
-            </tr>
-          </thead>
-          <tbody>
-            <!-- Each Country -->
-            <tr ng-repeat="(country, data) in countries">
-              <td>{{country}}</td>
-              <td>
-                <span ng-if="data" class="arrow up"></span>
-                <span ng-if="!data" class="arrow down"></span>
-              </td>
-            </tr>
-          </tbody>
-        </table>
 
-        <div id="terror-container">
-          <h3>Terror</h3>
+<div id="top-container">
+  <!-- Round Information -->
+  <div id="round-container">
+    <h2>Turn {{round}}</h2>
 
-          <div class="progress-wrapper">
-            <div class="progress-legend">{{terror}} / 250</div>
-            <div round-progress
-                max="250"
-                current="terror"
-                color="#9f0d33"
-                bgcolor="#399499"
-                radius="100"
-                stroke="20"
-                semi="false"
-                rounded="false"
-                clockwise="true"
-                responsive="false"
-                duration="800"
-                animation="easeInOutQuart"></div>
-          </div>
-        </div>
-      </div>
+    <div id="timer">
+      <h3 ng-if="!paused">
+        <timer countdown="roundDuration" max-time-unit="'minute'" interval="1000">{{mminutes}}:{{sseconds}}</timer>
+      </h3>
+      <h3 ng-if="paused">(Paused)</h3>
     </div>
+  </div>
 
-    <div class="centre-column">
-      <div id="round-container">
-        <h2 ng-if="!paused">
-          <timer countdown="roundDuration" max-time-unit="'minute'" interval="1000">{{mminutes}}:{{sseconds}}</timer>
-        </h2>
+  <!-- Terror Tracker -->
+  <div id="terror-tracker">
+    <h3>Global Terror: {{terror}} / 250</h3>
+    <div class="terror-border">
+      <div class="terror-bar" ng-class="getTerrorColour()" ng-style="getTerrorWidth()"></div>
+    </div>
+  </div>
+</div>
 
-        <!-- This needs to be formatted for use -->
+  <!-- Control Messages -->
+  <div id="control-messages">
+    <p>{{controlMessage}}</p>
+  </div>
 
-        <h3>Turn {{round}}</h3>
-        <h3 class="paused" ng-if="paused">(Paused)</h3>
-      </div>
 
-      <h3>News:</h3>
+  <!-- News -->
+  <div id="news-block">
 
-      <div class="news-container" ng-show="news.length>0" ng-repeat="news_item in news" ng-class="{ active: $index == 0 }">
-        <h4>{{news_item.title}}</h4>
-        <img ng-show="{{news_item.media_url.length>0}}" ng-src="{{news_item.media_url}}" />
-        <p>{{news_item.content}}</p>
-        <p class="date">{{news_item.created_at| date:'h:mma'}}</p>
-      </div>
+    <div id="breaking-block">
+      <h3>Breaking News</h3>
+      <div class="triangle"></div>
+    </div>
 
-      <div class="news-container active" ng-if="news.length==0">
-        <h4>Nothing has happened yet<h4>
+    <div id="news-image" ng-class="news[newsIndex].media_landscape == true ? 'landscape' : 'portrait'" ng-if="news[newsIndex].media_url">
```

Comment:
If we have padding vertically, we should just shrink the image viewer to fit instead of showing the padding.  Unless this is really difficult, then I'd just settle for centering.